dev/tools/localizations_utils.dart

```diff
@@ -24,7 +24,10 @@ class LocaleInfo implements Comparable<LocaleInfo> {
   /// and country is 2-3 characters and all uppercase.
   ///
   /// 'language_COUNTRY' or 'language_script' are also valid. Missing fields will be null.
-  factory LocaleInfo.fromString(String locale, {bool assume = false}) {
+  ///
+  /// When `deriveScriptCode` is true, the [scriptCode] will be tentatively derived from
```

Comment:
Tentatively? What would override the tentative value?

It looks like deriveScriptCode indices that the scriptCode should be computed from the locale string if it wasn't specified.

Reply:
you're right. This wasn't very clear. Reworded